_posts/2023-06-20-envoimoinscher.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: "[CVE-2023-30151] Improper neutralization of SQL parameters in the Boxtal (envoimoinscher) module from Boxtal for PrestaShop"
+categories: module
+author:
+- Profileo.com
+- Friends-Of-Presta.org
+meta: "CVE,PrestaShop,envoimoinscher"
+severity: "critical (9.8)"
+---
+
+In the Boxtal (envoimoinscher) module from Boxtal for PrestaShop, after version 3.1.10, a SQL injection vulnerability allows remote attackers to execute arbitrary SQL commands via the `key` parameter in the `ajax.php` front controller.
+
+## Summary
+
+* **CVE ID**: [CVE-2023-30151](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-30151)
+* **Published at**: 2023-06-20
+* **Advisory source**: Friends-Of-Presta
+* **Platform**: PrestaShop
+* **Product**: envoimoinscher
+* **Impacted release**: > 3.1.10,<= 3.3.8 (latest version, not fixed)
```

Comment:
(latest version, not fixed, *deprecated module to remove* or replace by *BoxtalConnect*)